argus/config/local.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016 Cloudbase Solutions Srl
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""Config options available for the local setup."""
+
+from oslo_config import cfg
+
+from argus.config import base as conf_base
+
+
+class LocalOptions(conf_base.Options):
+
+    """Config options available for the local setup."""
+
+    def __init__(self, config):
+        super(LocalOptions, self).__init__(config, group="local")
+        self._options = [
+            cfg.StrOpt(
+                "ip", default="", required=True,
+                help="The id of the flavor that is to be used."),
+            cfg.StrOpt(
+                "username", default="CiAdmin", required=True,
+                help="The default username existing on Argus-CI images used "
+                     "for connecting and interacting with the instance."),
+            cfg.StrOpt(
+                "password", default="Passw0rd", required=True,
```

Comment:
Just struck me that we are not following best practices right here, having a possible login password as a default value.
Seems better to simply add it in the config file, just a future suggestion.